appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -1267,6 +1266,30 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
+	def 'Try get raw activities with an invalid date'()
```

Comment:
Good to have an acceptance test. We could also add a unit test for `DayActivityDto.parseDate`.

Reply:
Good point. Added it.